src/core/StelTranslator.cpp

```diff
@@ -146,18 +146,18 @@ QString StelTranslator::tryTranslateChineseStar(const QString& s, const QString&
 		}
 		if (tens == 1)
 		{
-			number += chars[0];
+			number += QChar(chars[0]);
 		}
 		if (units)
 		{
 			number += chars[units];
 		}
 	}
-	const auto extra = match.captured(3);
+	const QString extra = match.captured(3);
 	if (!addedPresent) return translatedConstellation + number + extra;
 
-	const auto& translatedAdded = qtranslate(" Added", "chinese skycultures");
-	return QString("%1%2%3%4").arg(translatedConstellation, translatedAdded, number, extra);
+	const QString& translatedAdded = qtranslate("Added", "chinese skycultures");
+	return translatedConstellation + " " + translatedAdded + number + extra;
```

Comment:
It wasn't "missing", this is intended. Some locales (e.g. German) want a space here, others (e.g. Chinese, or even Russian) don't.

Reply:
Ah, OK. Then this must be annotated in the typical // TRANSLATORS:... comment and be visible in Transifex.

Reply:
What to do that translators will see the note that the space is important?

Reply:
Please see https://github.com/Stellarium/stellarium/commit/ebc48cb876ab69a8329ae091d924a1c7f3a57d63